### ISSUE TITLE
e-mail повідомлення.

### ISSUE THREAD

**Issue by p-butovets** [mblog-dev/wish#1]
Непогано було б, якщо я міг би настроїти, яки саме повідомлення я хочу отримувати на почту, бо зараз на почту сиплятся абсолютно всі повідомлення і вимкнути чи відписатись це я не можу:(

**Comment by KostyaTretyak** [comment]
Так, я якраз над цим зараз працюю. Вибачаюсь за тимчасовий спам.

А які саме опції ви хотіли бачити для налаштування підписки? Планується, як мінімум, відписатись від усіх коментарів у темі.

**Comment by KostyaTretyak** [comment]
Так, здається мінімум зробив. Вже повинен приходити лінк із можливістю відписатись від теми повністю.

**Comment by p-butovets** [comment]
Дякую, відписався. Яки налаштування повідомлень робити, то вже мабуть до користувачив треба звернутись згодом.

**Comment by KostyaTretyak** [comment]
Не бачу відписку у таблиці. Ви впевнені, що то відписка була із сайту mblog.dev?

**Comment by p-butovets** [comment]
Можливо помилився, ще раз відписався:) Є ідея: аби при створенні посту я міг обрати, чи підписатись мені на оновлення цього посту.

**Comment by KostyaTretyak** [comment]
О, тепер бачу.

**Comment by KostyaTretyak** [comment]
> Є ідея: аби при створенні посту я міг обрати, чи підписатись мені на оновлення цього посту.

Взагалі то дивно, що автор не хоче читати коментарі під своєю публікацією. Підкажіть які можуть бути причини такого бажання.

**Comment by p-butovets** [comment]
На моєму прикладі: я запостив, і хочу прочитати, але все разом можливо ввечері, після роботи або на перерві. І кожен коммент на почті мене дуже відволікав
